sharepoint/sharepoint-ps/sharepoint-online/Set-SPOContainerTypeConfiguration

```diff
@@ -0,0 +1,81 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/Set-SPOContainerTypeConfiguration
+applicable: SharePoint Online
+title: Set-SPOContainerTypeConfiguration
+schema: 2.0.0
+author: FarreltinF
+ms.author: FarreltinF
```

Comment:
>FarreltinF [](http://example.com/codeflow?start=11&length=10)

the `ms.author` one should stay as your MS alias -- only the `author` value needed to change to GitHub username